criteria/common/src/org/immutables/criteria/matcher/OrderMatcher.java

```diff
@@ -27,14 +27,14 @@
   /**
    * Sort current expression in <i>ascending</i> order.
    */
-  default Ordering asc() {
+  default Collation asc() {
```

Comment:
Is it possible to keep return type of `asc()` as is (ie `Collation`) ? While it doesn't add functionality it hides all methods from user during IDE auto-complete.

Reply:
sure, it was needed in test, I will cast for now